docs/hub/adding-a-library.md

```diff
@@ -246,7 +246,8 @@ The metadata held in model cards is the best way to supercharge your model. It i
 define tags for your library or framework, the type of model uploaded, the language, license, evaluation results,
 and more.
 
-The full model card specification can be seen below:
+The full model card specification is  [here](https://github.com/huggingface/huggingface_hub/blame/main/modelcard.md), it's also repeated below for convenience:
```

Comment:
Do we really want to maintain the spec in two places? Maybe we can delete the one below to avoid obsolete content

Reply:
yes i think it makes sense

Reply:
i'll leave it for now but we can remove it later